tests/README.md

```diff
@@ -142,3 +142,16 @@ To run specific tests inside a suite:
 ```console
 _output/tests/linux_amd64/integration -test.v -test.timeout 1800s -test.run CephSmokeSuite -testify.m TestARookClusterInstallation_SmokeTest
 ```
+
+### To run tests on OpenShift environment
+
+- You have to setup OpenShift environment by your own and export KUBECONFIG before executing the tests.
+- Make sure `oc` executable file is in your PATH.
+- Currenttly we are enabling only `CephSmokeSuite`  on OpenShift environment.
+
+
```

Comment:
How about documenting all the test env vars here?

Reply:
Done.